src/tests/ftest/deployment/server_rank_failure.py

```diff
@@ -176,43 +237,42 @@ def verify_rank_failure(self, ior_namespace, container_namespace):
         self.log.info("Waiting 5 sec for IOR to start writing data...")
         time.sleep(5)
 
-        # 3. While IOR is running, kill daos_engine from two of the ranks.
+        # 7. While IOR is running, kill daos_engine from two of the ranks.
         # Arbitrarily select node index, say index 1.
-        self.kill_engine(engine_kill_host=self.hostlist_servers[1])
+        self.kill_engine(engine_kill_host=engine_kill_host)
 
         # Wait for IOR to complete.
         ior_thread.join()
 
-        # 4. Verify that IOR failed.
-        errors = []
-        self.log.info("----- IOR results 1 -----")
+        # 8. Verify that IOR failed.
+        self.log.info("----- IOR results 2 -----")
         self.log.info(ior_results)
         if job_num not in ior_results or ior_results[job_num][0]:
             errors.append("First IOR didn't fail as expected!")
 
-        # 5. Restart daos_servers. It's not easy to restart only on the host where the
+        # 9. Restart daos_servers. It's not easy to restart only on the host where the
         # engines were killed, so just restart all.
         self.restart_all_servers()
 
-        # 6. Verify the system status by calling dmg system query.
+        # 10. Verify the system status by calling dmg system query.
         output = self.get_dmg_command().system_query()
         for member in output["response"]["members"]:
             if member["state"] != "joined":
                 errors.append(
                     "Server rank {} state isn't joined!".format(member["rank"]))
 
-        # 7. Verify that the container Health is HEALTHY.
+        # 11. Verify that the container Health is HEALTHY.
         if not self.check_container_health(
                 container=self.container, expected_health="HEALTHY"):
             errors.append("Container health isn't HEALTHY after server restart!")
 
-        # 8. Run IOR and verify that it works.
-        job_num = 2
+        # 12. Run IOR and verify that it works.
+        job_num = 3
         self.run_ior_report_error(
-            job_num=job_num, results=ior_results, file_name="test_file_2",
+            job_num=job_num, results=ior_results, file_name="test_file_3",
             pool=self.pool, container=self.container, namespace=ior_namespace)
 
-        self.log.info("----- IOR results 2 -----")
+        self.log.info("----- IOR results 3 -----")
```

Comment:
If you wanted to be less error prone...
```suggestion
        self.log.info("----- IOR results %s -----", job_num)
```

Reply:
Yeah I was thinking about that :)